feat(dashboard): Optimize layout and update EMQX logo

ysfscream opened the pull request:
### What does this PR do?

This PR enhances the EMQX integration for Datadog by improving the dashboard layout, updating the EMQX logo, and enhancing overall visual consistency. Specifically, it:

- Optimizes the dashboard panel layout and design for better user experience
- Updates the EMQX logo to align with our latest brand identity
- Improves the overall visual consistency of the integration

### Motivation

As an official developer and maintainer of EMQX, we are committed to providing the best possible experience for users of our Datadog integration. This update aims to enhance the visual appeal and usability of the EMQX dashboard within Datadog, ensuring it reflects our current branding and provides a more intuitive interface for monitoring EMQX deployments.

### Review checklist

- [x] PR has a meaningful title
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

I can confirm that these changes are in line with our current branding guidelines and UX best practices. The updated dashboard layout has been tested internally with various EMQX deployment scenarios to ensure it provides valuable insights at a glance.

dkirov-dd commented on emqx/assets/dashboards/overview.json:
```diff
@@ -1,1578 +1 @@
-  "reflow_type": "fixed"
-}
\ No newline at end of file
+{"title":"EMQX Overview (cloned by ysf)","description":"This dashboard provides an overview of your EMQX environment. It monitors node status, provides information on message status, and provides insight into various other EMQX setup observability measures. (cloned)","widgets":[{"id":7207015794187642,"definition":{"title":"EMQX","banner_img":"https://raw.githubusercontent.com/ysfscream/images/mai ...
```
I think you've missed running `jq` to format

emarsha94 commented on emqx/CHANGELOG.md:
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG - EMQX
 
+## 1.2.0 / 2024-07-19
```
```suggestion
## 1.2.0 / 2024-08-13
```

emarsha94 commented:
Waiting on logo to be published on our end then will merge!

emarsha94 commented:
Please update you branch and then we can merge

ysfscream commented:
@emarsha94 Updated.

emarsha94 commented:
hey @ysfscream Could you run "ddev validate config emqx -s"? This will fix any other issues you have in the files. Once that is done, the last validation should pass!!

ysfscream commented:
@emarsha94 sorry,  but I don't know how to run `ddev validate config emqx -s` locally. Could you please give me a hand? Thank you.

emarsha94 commented:
Hi @ysfscream you should have ddev installed on your machine already since you opened the PR. If not, you will need to install ddev and pull down this branch and run the command. We are unable to do this for you since this is local to your machine and branch

ysfscream commented:
I don't know why. It has run successfully locally for me, but CI still has an error.

<img width="973" alt="image" src="https://github.com/user-attachments/assets/6a479ffe-b870-4b42-a69d-75337e66eaf6">

<img width="707" alt="image" src="https://github.com/user-attachments/assets/2b4e4fff-1b36-4d58-903c-01089f77bfb2">

emarsha94 commented:
> I don't know why. It has run successfully locally for me, but CI still has an error.
> 
> <img alt="image" width="431" src="https://private-user-images.githubusercontent.com/21299158/370654227-f1f0db14-fc5c-4bea-859f-238ca61d9ba4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyOTY5ODcsIm5iZiI6MTcyNzI5NjY4NywicGF0aCI6Ii8yMTI5OTE1OC8zNzA2NTQyMjctZjFmMGRiMTQtZmM1Yy00YmVhLTg1OWYtMjM4Y2E2MWQ5YmE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDIwMzgwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmMWM0ZDdhN2JjMjMxNzFmMzUwOTJhOGVmNGI4NGZmY2Q5ZmJiMjE1NGY1NjViYmU1MThlMjMxMTQ4ZjgxN2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.MIZGtl7sVjV2CkUs9ntU-DPQ4UfyC5FW-7UsIyH-Hso"> <img alt="image" width="707" src="https://private-user-images.githubusercontent.com/21299158/370653733-2b4e4fff-1b36-4d58-903c-01089f77bfb2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyOTY5ODcsIm5iZiI6MTcyNzI5NjY4NywicGF0aCI6Ii8yMTI5OTE1OC8zNzA2NTM3MzMtMmI0ZTRmZmYtMWIzNi00ZDU4LTkwM2MtMDEwODlmNzdiZmIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDIwMzgwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4NWU5ODE2NmE5NTE5YTdmMmFhODU1NTI3MmFmZDEyMDQyNGRjN2IyMTdhMDk4YWJmMDdlZWU0ZmU3YWUyZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.OAks0mVD8Y67Zs8Olqjwnce4I4Rb0eouKJ5_5psS0iQ">

Can you try running the command: ddev validate config emqx -s 

You may need to update your ddev version if that doesn't work,

ysfscream commented:
I run it on my screenshot...

<img width="508" alt="image" src="https://github.com/user-attachments/assets/8d2fb986-19ad-474e-93cf-6261f1d3cc22">

This is the latest version...

<img width="314" alt="image" src="https://github.com/user-attachments/assets/6fa4fd89-199b-452c-9d24-0d3e838ac448">

<img width="685" alt="image" src="https://github.com/user-attachments/assets/57603cb3-7c8b-4a89-a5c0-9ac968ad7ad3">

emarsha94 commented:
Hey @ysfscream After running the command, did you commit the branch now that the files are in sync? If not, can you try committing the updates and see if that resolves the validation?

ysfscream commented:
@emarsha94 Sorry, we just updated the PR now. So far, there is no problem with CI, and I have also updated the branch code. If there is no problem, please help merge the code. Thank you!

ysfscream commented:
Hi, team. any updates?
